test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -114,6 +115,27 @@ def test_dist_operations_fastai_gpu(sagemaker_session, framework_version, ecr_im
     _assert_s3_file_exists(sagemaker_session.boto_region_name, model_s3_url)
 
 
+@pytest.mark.processor("gpu")
+@pytest.mark.integration("pt_s3")
```

Comment:
nit: Please use the same (or similar) `integration` pytest marker for all the plugin tests (i.e., the EC2 test, the sanity test, and the SM test).

Reply:
done